Documentation/docs/source/conf.py

```diff
@@ -75,6 +80,39 @@
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
+# -- Sphinx gallery configuration -----------------------------------------
+sphinx_gallery_conf = {
+    # path to your examples scripts
+    'examples_dirs': ['user_guide_source'],
+    # path where to save gallery generated examples
+    'gallery_dirs': ['user_guide'],
+    'filename_pattern': 'guide.py'
+}
+
+# Copy rst in source dirs to build dir
+for i in range(len(sphinx_gallery_conf['examples_dirs'])):
+    gallery_dir = sphinx_gallery_conf['gallery_dirs'][i]
+    source_dir = sphinx_gallery_conf['examples_dirs'][i]
+    # Create gallery dirs if it doesn't exist
+    try:
+        os.mkdir(gallery_dir)
+    except FileExistsError:
```

Comment:
Change FileExistsError to OSError so the docs can be built with Python 2.  FileExistsError is only available in Python 3.